config/locales/admin/participants/change_relationship.yml

```diff
@@ -0,0 +1,9 @@
+en:
+  admin:
+    participants:
+      change_relationship:
+        reason_for_change:
+          options:
+            wrong_programme: "The participant was placed in the wrong training programme."
+            change_of_circumstances: "The participant was originally placed in the right training programme, but their circumstances have changed."
```

Comment:
A quick suggestion: we could avoid the repetition of "The participant" by switching to "They"?

```suggestion
            wrong_programme: "They were placed in the wrong training programme."
            change_of_circumstances: "They were originally placed in the right training programme, but their circumstances have changed."
```

Could also include the name of the teacher in the question to make it clearer?

<img width="904" alt="Screenshot 2023-07-05 at 16 08 45" src="https://github.com/DFE-Digital/early-careers-framework/assets/30665/27b2434b-b6e8-4afe-8d7c-aa702b245030">

Reply:
Much nicer 👍